app/services/action_queue.rb

```diff
@@ -148,7 +149,8 @@ def transaction_payload
             currency:     data[:currency]
           }.merge(fake_card_info),
           action: {
-            source: data[:source]
+            source: data[:source],
+            skip_confirmation: 1
```

Comment:
Currently, GoCardless sends two confirmation emails already when someone creates a new direct debit (we might work on that in the future, but not right now). As a result, the donor effectively already has a receipt and they should not get an ActionKit confirmation email (if set up) as a third email. So if this code works as expected, it will pass skip_confirmation=1 to ActionKit for a direct debit donation.